specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2015-11-01-preview/examples/LinkedServicesListByWorkspace.json

```diff
@@ -0,0 +1,30 @@
+{
+   "parameters":{
+      "resourceGroupName":"mms-eus",
+      "workspaceName":"TestLinkWS",
+      "api-version":"2015-11-01-preview",
+      "subscriptionId":"a6383be3-f0e8-4968-93d5-10f2625f5bb"
+   },
+   "responses":{
+      "200":{
+         "body":[
+            {
+               "properties":{
+                  "resourceId":"/subscriptions/a6383be3-f0e8-4968-93d5-10f2625f5bb5/resourceGroups/mms-eus/providers/Microsoft.Automation/automationAccounts/TestAccount"
+               },
+               "id":"/subscriptions/a6383be3-f0e8-4968-93d5-10f2625f5bb5/resourcegroups/mms-eus/providers/microsoft.operationalinsights/workspaces/testlinkws/linkedservices/automation",
+               "name":"TestLinkWS/Automation",
+               "type":"Microsoft.OperationalInsights/workspaces/linkedServices"
+            },
+            {
+               "properties":{
+                  "resourceId":"/subscriptions/a6383be3-f0e8-4968-93d5-10f2625f5bb5/resourceGroups/mms-eus/providers/Microsoft.Automation/automationAccounts/TestAccount2"
+               },
+               "id":"/subscriptions/a6383be3-f0e8-4968-93d5-10f2625f5bb5/resourcegroups/mms-eus/providers/microsoft.operationalinsights/workspaces/testlinkws/linkedservices/automation",
```

Comment:
If these are real subscription IDs, they should be obscured.

Reply:
Changed to 00000000-0000-0000-0000-00000000000